tests/Browser/UserTest.php

```diff
@@ -0,0 +1,204 @@
+<?php
+
+namespace Tests\Browser;
+
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Laravel\Dusk\Browser;
+use Tests\DuskTestCase;
+use App\User;
+
+class UserTest extends DuskTestCase
+{
+    /**
+     * Permisson account admin
+     *
+     * @return $admin
+     */
+    public function accountAdmin()
+    {
+        $admin = User::where('email', 'loitq@lifull-tech.vn')->first();
+
+        return $admin;
+    }
+
+    /**
+     * Get data from config to test.
+     *
+     * @return array
+     */
+    public function getDataTest()
+    {
+        return config('constants');
+    }
+
+    /**
+     * Admin logged go to dashboard
+     * case 1-1
+     *
+     * @return void
+     */
+    public function testLoginToPageAdmin()
+    {
+        $admin = $this->accountAdmin();
+        $data = $this->getDataTest();
+
+        if (!$admin) {
```

Comment:
In the test case condition check for create data , if you want check -> preparation , please use setUpBeforeClass

Reply:
I understand, created data in the function accountAdmin